austrakka/components/iam/role/access/__init__.py

```diff
@@ -0,0 +1,40 @@
+from austrakka.components.iam.role.access.funcs import get_access, add_access, remove_access
+from austrakka.utils.cmd_filter import hide_admin_cmds
+from austrakka.utils.options import *
+from austrakka.utils.output import table_format_option
+
+
+@click.group()
+@click.pass_context
+def access(ctx):
+    """Commands related to access definition for roles."""
+    ctx.context = ctx.parent.context
+
+
+@access.command('get', hidden=hide_admin_cmds())
```

Comment:
Do you think `access get` is clear enough? had to think about it for a minute. But maybe that's my dev brain thinking of other words.

Reply:
Rename to role_access_get.